Introduction: contributions and paper structure (#59)
MrLongbottom/P9-Rapport: sections/introduction/contributions.tex

These questions also emphasize the contributions this paper brings.
We implement a method for creating an adjacency matrix for running \gls{pr}, with no previous connections given between the nodes.
This is done by calculating the topic similarity between each document, using Jensen-Shannon distance, and using these as the elements in the matrix.

> We also suggest ways to calculate the performance an information retrieval model has at both finding specific documents and documents that have a general topic.

Reviewer: ```suggestion
We also suggest ways to calculate the performance of a \gls{ir} model which has to find both specific documents and documents that have a general topic.
```
